wp-content/themes/reconasia/inc/template-tags.php

```diff
@@ -249,7 +249,7 @@ function reconasia_display_categories() {
 		}
 
 		/* translators: used between list items, there is a space after the comma */
-		$categories_list = get_the_category_list(', ');
+		$categories_list = get_the_category_list(' ');
```

Comment:
```suggestion
		$categories_list = get_the_category_list();
```

If we remove the empty space from the `get_the_category_list()` function, we'll get an unordered list, which is more semantically correct implementation of this. See the [documentation](https://developer.wordpress.org/reference/functions/get_the_category_list/) for details.